content/legacy/_pub_2001_09_26_crypto1.md

```diff
@@ -285,7 +285,7 @@ Meanwhile, if Eve has sniffed the network traffic to intercept the message and s
 <br />
 • <a href="http://www.ietf.org/rfc/rfc2437.txt">Kaliski &amp; Staddon, PCKS #1, RSA Cryptography Specifications</a><br />
 <br />
-• <a href="http://www.rsasecurity.com/rsalabs/pkcs/pkcs-3/">PKCS #3, Diffie-Hellman Key Agreement Standard</a><br />
+• <a href="http://web.archive.org/web/20061207062720/http://www.rsasecurity.com:80/rsalabs/pkcs/pkcs-3/">PKCS #3, Diffie-Hellman Key Agreement Standard</a><br />
```

Comment:
Update this one to https://www.teletrust.de/fileadmin/files/oid/oid_pkcs-3v1-4.pdf